local-volume/README.md

```diff
@@ -17,7 +17,7 @@ directories by creating and cleaning up PersistentVolumes for each volume.
 
 ## Feature Status
 
-Current status: 1.7 & 1.8 - Alpha
+Current status: 1.7, 1.8, & 1.9 - Alpha
```

Comment:
Can you leave out 1.9 for now.  I'm going to update this soon with 1.9 status; a lot has changed.